uvm/js/common/util/Converter.js

```diff
@@ -23,6 +23,18 @@ Ext.define('Ung.util.Converter', {
         return Ext.util.Format.date(date, 'timestamp_fmt'.t());
     },
 
+    convertFn: function (value) {
```

Comment:
Can you please add some documentation for this function ?

Reply:
Also, let's rename it to something like "convertDate" instead of this generic name.

Reply:
changed the convertFn name to convertDate

Reply:
@mayur-dhande  will do that in next PR.